hash_practice/exercises.py

```diff
@@ -2,18 +2,43 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n log n)

```

Comment:
👍   The time complexity is right if the words are all of limited length (like english words). 

The space complexity is O(n) because you're building a dictionary of n words.